purchases/src/main/kotlin/com/revenuecat/purchases/models/Period.kt

```diff
@@ -31,9 +31,9 @@ data class Period(
         private const val DAYS_PER_WEEK = 7.0
         private const val DAYS_PER_MONTH = 30.0
         private const val DAYS_PER_YEAR = 365.0
-        private const val WEEKS_PER_MONTH = 4.0
         private const val WEEKS_PER_YEAR = 52.14
```

Comment:
Should we also calculate this one so we have the same precision?

Reply:
I was shooting for "least amount of changes possible" but we could.

It'd be nice to update them all really to also take into account leap years

Reply:
Will make that change separately.